src/components/ContentWrapper.jsx

```diff
@@ -280,18 +300,25 @@ const ContentWrapper = (props) => {
   const menuItemRender = ({
     path, icon, name, disableIfNoExperiment, disabledByPipelineStatus,
   }) => {
-    const noExperimentDisable = !experimentId ? disableIfNoExperiment : false;
+    const pipelinesCompleted = () => {
+      if (pipelineStatusKey === pipelineStatus.SUCCEEDED
+        && gem2sStatusKey === pipelineStatus.SUCCEEDED) {
+        return true;
+      }
+      return false;
+    };
+    const notProcessedExperimentDisable = disableIfNoExperiment && !routeExperimentId && !pipelinesCompleted();
```

Comment:
Can you put a space before and after `notProcessedExperimentDisable` to make it easier to read?